src/EnergyPlus/DaylightingManager.cc

```diff
@@ -2828,22 +2827,22 @@ Real64 CalcObstrMultiplier(EnergyPlusData &state,
     SkyGndUnObs = 0.0;
 
     // Tuned Precompute Phi trig table
-    if (AltSteps != AltSteps_last) {
+    if (AltSteps != state.dataDaylightingManager->AltSteps_last) {
         for (int IPhi = 1, IPhi_end = (AltSteps / 2); IPhi <= IPhi_end; ++IPhi) {
             Phi = (IPhi - 0.5) * DPhi;
             state.dataDaylightingManager->cos_Phi(IPhi) = std::cos(Phi);
             state.dataDaylightingManager->sin_Phi(IPhi) = std::sin(Phi);
         }
-        AltSteps_last = AltSteps;
+        state.dataDaylightingManager->AltSteps_last = AltSteps;
```

Comment:
I messed something up here. Not sure why this is showing up as a diff.

Reply:
It looks like AltSteps_last is now moved into state, where it was local before.  Is that right?